spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskDefinitionController.java

```diff
@@ -113,9 +131,50 @@ public TaskDefinitionResource save(@RequestParam("name") String name,
 	}
 
 	/**
+	 * Register a composed task definition for future execution.
+	 *
+	 * @param name of the composed task
+	 * @param graph DSL definition for the composed task
+	 */
+	@RequestMapping(value = "/compose", method = RequestMethod.POST)
```

Comment:
Understood about the controller.  But for the resource I'm not so sure.  It is still just a task that is created.